### PR TITLE
Rugby live scores

### PR DESCRIPTION
**Pull request by jennysivapalan**
PR contains:
- Parsing of the XML match live score from Opta.
- Creates an agent that will hold the rugby lives scores endpoint. (This endpoint only shows the live scores of the current competition (currently hardcoded to friendlies).)
- Creates HTML and Json endpoints that will fetch live score for a given match (`/rugby/api/live-score/:year/:month/:day/:homeTeamId/:awayTeamId` where `homeTeamId` and `awayTeamId` are the IDs of the teams supplied in the feeds).

screen shot 2015-08-19 at 15 17 36

To do:
- Call the `/rugby/api/live-score/:year/:month/:day/:homeTeamId/:awayTeamId` to enhance the match page. I tried following the football implementation but I need to pair with someone familiar with the JS code (cc @rich-nguyen, @OliverJAsh ). This can be separated out to a separate PR.
- ~~Provision the properties - which I will work on now.~~ Done

### CODE REVIEW

File: standalone/conf/standalone.routes [hunk 1]
```diff
@@ -125,6 +125,9 @@ GET        /football/match/:matchId
 GET        /football/match-redirect/:year/:month/:day/:homeTeamId/:awayTeamId                  football.controllers.MoreOnMatchController.redirectToMatch(year,month,day,homeTeamId,awayTeamId)
 GET        /football/match-redirect/:matchId                                                   football.controllers.MoreOnMatchController.redirectToMatchId(matchId)
 
+GET        /rugby/api/live-score/:year/:month/:day/:homeTeamId/:awayTeamId.json                rugby.controllers.LiveScoresController.liveScoreJson(year, month, day, homeTeamId, awayTeamId)
+GET        /rugby/api/live-score/:year/:month/:day/:homeTeamId/:awayTeamId
```
Comment (rich-nguyen) [inline standalone/conf/standalone.routes]: controller!

File: sport/app/rugby/feed/OptaFeed.scala [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+object OptaFeed extends ExecutionContexts with Logging {
+  private val xmlContentType = ("Accept", "application/xml")
+
+  private def getLiveScoresResponse: Future[String] = {
+
+    val endpoint = conf.Configuration.optaRugby.endpoint
+    endpoint.map { e =>
+      WS.url(e)
+        .withHeaders(xmlContentType)
+        .withQueryString("competition" -> "3", "season_id" -> "2016", "psw" -> conf.Configuration.optaRugby.apiKey.getOrElse(""),
```
Comment (rich-nguyen) [inline sport/app/rugby/feed/OptaFeed.scala]: these magic strings would be great to have as `val` s for easy reading/tweaking.

**Comment by rich-nguyen** [comment]
:+1:

**Comment by OliverJAsh** [comment]
Oooh exciting! Some of my first proper Scala code :smile:

File: dev-build/conf/routes [hunk 3]
```diff
@@ -181,6 +181,9 @@ GET         /football/match/:matchId
 GET         /football/match-redirect/:year/:month/:day/:homeTeamId/:awayTeamId                                       football.controllers.MoreOnMatchController.redirectToMatch(year,month,day,homeTeamId,awayTeamId)
 GET         /football/match-redirect/:matchId                                                                        football.controllers.MoreOnMatchController.redirectToMatchId(matchId)
 
+GET         /rugby/api/live-score/:year/:month/:day/:homeTeamId/:awayTeamId.json                                     rugby.controllers.LiveScoresController.liveScoreJson(year, month, day, homeTeamId, awayTeamId)
+GET         /rugby/api/live-score/:year/:month/:day/:homeTeamId/:awayTeamId                                          rugby.controllers.LiveScoresController.liveScore(year, month, day, homeTeamId, awayTeamId)
```
Comment (gklopper) [inline dev-build/conf/routes]: this will need a "router" route

**Pull request merged**